src/main/java/frc/robot/subsystems/conveyor/ConveyorIOSim.java

```diff
@@ -36,7 +36,7 @@ public void setVelocity(MutableMeasure<Velocity<Angle>> velocity) {
     @Override
     public void updateInputs() {
         conveyor.update(Timer.getFPGATimestamp());
-        inputs.velocity.mut_replace(Units.RotationsPerSecond.of(conveyor.getVelocity()));
+        inputs.currentVelocity.mut_replace(Units.RotationsPerSecond.of(conveyor.getVelocity()));
         inputs.appliedVoltage.mut_replace(Units.Volts.of(conveyor.getBusVoltage()));
         inputs.appliedCurrent.mut_replace(Units.Amps.of(conveyor.getOutputCurrent()));
```

Comment:
Like in the intake, better to supply a double and the units instead of creating an object every loop.

Reply:
fe653a428065d4ed023b41a4f1444dcdcf39c64f